packages/url_launcher/url_launcher/README.md

```diff
@@ -168,14 +195,17 @@ original APIs.
 We recommend checking first whether the directory or file exists before calling `launchUrl`.
 
 Example:
+
+<?code-excerpt "files.dart (file)"?>
 ```dart
-var filePath = '/path/to/file';
+final String filePath = testFile.absolute.path;
 final Uri uri = Uri.file(filePath);
 
-if (await File(uri.toFilePath()).exists()) {
-  if (!await launchUrl(uri)) {
-    throw 'Could not launch $uri';
-  }
+if (!File(uri.toFilePath()).existsSync() || !await canLaunchUrl(uri)) {
```

Comment:
We've moved pretty strongly away from recommending `canLaunchUrl` before `launchUrl` since it most cases—as here—it's actually useless. I'm fine with this new structure, but the second condition should be removed.

Reply:
Removed the call to `canLaunchUrl`, and the note about not having a registered handler.